src/sdk-reference/cpp/1/getting-started/snippets/document.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+
+#include "kuzzle.hpp"
+#include "websocket.hpp"
+
+using kuzzleio::Kuzzle;
+
+#define K_INDEX_NAME "nyc-open-data"
+#define K_COLLECTION_NAME "yellow-taxi"
+
+
+int main(int argc, char * argv[]) {
+    std::string res;
+    // Instanciate a Kuzzle client
+    // with a WebSocket connection.
+    // Replace "kuzzle" with
+    // your Kuzzle hostname like "localhost"
+    Kuzzle *k = new Kuzzle(new kuzzleio::WebSocket("kuzzle"));
```

Comment:
`k` looks like a Go variable name. In c++ variable are more explicit, like `kuzzle`

Reply:
@etrousset